tests/smoke/alp_cholesky.cpp

```diff
@@ -200,11 +267,15 @@ void alp_program( const inpdata &unit, alp::RC &rc ) {
 		return;
 	}
 
-	rc = rc ? rc : algorithms::cholesky_uptr( L, H, ring );
+ 	rc = rc ? rc : algorithms::cholesky_uptr( L, H, ring );
 #ifdef DEBUG
-	print_matrix( std::string(" << L >> "), L );
+ 	print_matrix( std::string(" << L >> "), L );
 #endif
-	rc = rc ? rc : check_cholesky_solution( H, L, ring );
+ 	rc = rc ? rc : check_cholesky_solution( H, L, ring );
+
+	// TODO
+	// rest of cholesky algorithms are not implemented for complex version
```

Comment:
I am a bit confused by the title of this review :P Does this PR enable computing Cholesky of Hermitian-PD matrices (saw some changes in the algorithms above that I thought were going in this direction) up to say here? In this case perhaps from here on it should be guarded (eg, something like `#ifndef _COMPLEX`)? Or is computing Cholesky of HPD matrices currently not possible and so this PR is meant for temporarily disabling this test?

Reply:
I implemented Cholesky of Hermitian-PD only for unblocked out-of-place version, because I need Cholesky for another algorithm. While it would be ideal to have to for all versions (blocked and in-place vesrions), it is currently blocked by issue #587. Once #587 is resolved, complex-Cholesky for blocked versions should be straightforward. Rather than adding macros, I disable check of complex-Cholesky, and the implemented version is automatically checked by POTRI.